crud.py

```diff
@@ -0,0 +1,154 @@
+from datetime import date
+from typing import Type, List, Dict, Any, Sequence
+
+from sqlalchemy import select, Row, RowMapping
+from sqlalchemy.ext.asyncio import AsyncSession
+
+import models
+import schemas
+from sqlalchemy.orm import selectinload
+
+from models import Author, Book
+
+
+async def get_authors(db: AsyncSession, skip: int = 0, limit: int = 5) -> list[dict[str, list[dict[str, Any]] | Any]]:
+    stmt = (
+        select(models.Author)
+        .options(selectinload(models.Author.books))
+        .offset(skip)
+        .limit(limit)
+    )
+    result = await db.execute(stmt)
+    authors = result.scalars().all()
+    return [
+        {
+            "id": author.id,
+            "name": author.name,
+            "bio": author.bio,
+            "books": [
+                {
+                    "id": book.id,
+                    "title": book.title,
+                    "summary": book.summary,
+                    "publication_date": book.publication_date,
+                    "author_id": book.author_id,
+                }
+                for book in author.books
+            ],
+        }
+        for author in authors
+    ]
+
+
+async def get_author(db: AsyncSession, author_id: int) -> dict:
+    stmt = (
+        select(models.Author)
+        .options(selectinload(models.Author.books))
+        .where(models.Author.id == author_id)
+    )
+    result = await db.execute(stmt)
+    author = result.scalars().first()
+    return {
+        "id": author.id,
+        "name": author.name,
+        "bio": author.bio,
+        "books": [
+            {
+                "id": book.id,
+                "title": book.title,
+                "summary": book.summary,
+                "publication_date": book.publication_date,
+                "author_id": book.author_id,
+            }
+            for book in author.books
+        ],
+    }
```

Comment:
```suggestion
    return models.Author.from_orm(author)
```
have you tried this ?